packages/addon/src/menu.ts

```diff
@@ -100,6 +105,33 @@ export async function menuLogout() {
   // TODO: Implement proper menu item cleanup
   console.log('🧹this should clear the menu items');
   await browser.TBProMenu.clear('root');
+  await browser.storage.local.remove(STORAGE_KEY_AUTH);
```

Comment:
Logging out of hamburger logs out of add-on, part 1.

We begin by removing the user auth info from storage.